backend/api/Controllers/RobotController.cs

```diff
@@ -257,6 +257,12 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] CreateRobotQuery rob
             _logger.LogError(e, "{message}", message);
             return StatusCode(StatusCodes.Status500InternalServerError, message);
         }
+        catch (RobotPositionNotFoundException e)
+        {
+            string message = "A suitable robot position could not be found";
```

Comment:
"... for one or more of the desired tags"
Also this means that the entire mission will fail if one tag is missing, is this desired functionality?